src/main/resources/db/changelog/changes/11-add-id-deleted-column-for-accommodations.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-telegramId-column-to-user
+      author: Maksym
```

Comment:
You added soft delete column in acomodation, but it says smth about telegram id? Is this correct?